cmd/vegawallet/commands/cmd.go

```diff
@@ -20,7 +21,8 @@ import (
 )
 
 const (
-	DefaultForwarderRetryCount = 5
+	DefaultRequestRetryCount  = 5
+	DefaultMaxRequestDuration = 5 * time.Second
```

Comment:
```suggestion
	defaultRequestRetryCount  = 5
	defaultMaxRequestDuration = 5 * time.Second
```

I don't see these used outside of the current package.